_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -27,20 +27,19 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element with the [semantic role][] of [rowheader][] or [columnheader][] for which all of the following is true:
+This rule applies to any HTML element with the [semantic role][] of [rowheader][] or [columnheader][] within a [table][] or [grid][] element, where the [table][] or [grid][] element is [visible][].
```

Comment:
`[table]` and `[grid]` refer to ARIA roles (in the references list). I feel it's not super clear that "a table element" actually means "an element with a role of table".

Also, we normally backticks roles and element (``a [`table`][table] element``).